js/everything.js

```diff
@@ -495,11 +495,11 @@ $(function() {
                     $("#search-results").show();
                     // Clear out results
                     $("#search-results ul").html("");
-
-                    var start = "<li><h4><a href=?v=";
+                    var start = "<li><h4><a style='float:right' href=?v=";
                     var end = "</a></h4></li>";
                     $.each(data.items, function(index, result) {
-                        $("#search-results ul").append(start + result.id.videoId + ">" + result.snippet.title  + end);
+                        var thumbTag = "<img style='float:left; margin: auto' src='https://img.youtube.com/vi/" + result.id.videoId + "/1.jpg'>";
```

Comment:
I'd like to break this up into a couple of steps, make the image URL it's own var, etc to make this more readable.